README.md

```diff
@@ -94,3 +94,23 @@ $ cd build
 $ cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_PYTORCH_INCLUDE_PATHS="<PATH_PREFIX>/torch;<PATH_PREFIX>/torch/torch/csrc/api/include;<PATH_PREFIX>/torchvision" -DTRITON_PYTORCH_LIB_PATHS="<LIB_PATH_PREFIX>" ..
 $ make install
 ```
+
+## Using the OpenVINO Backend
+
+### Parameters
+
+Disabling the optimized execution of the PyTorch models is done through the Parameters section of the model's 'config.pbtxt' file.
+
+* `DISABLE_OPTIMIZED_EXECUTION`: Boolean flag to disable the optimized execution of TorchScript models. By default the optimized execuiton is always enabled.
```

Comment:
We should add a small note about why to use this flag. In ideal scenario, disabling optimized execution appears counter-intuitive.

Reply:
Done